src/cow-react/modules/limitOrders/containers/LimitOrdersConfirmModal/index.tsx

```diff
@@ -98,6 +102,21 @@ export function LimitOrdersConfirmModal(props: LimitOrdersConfirmModalProps) {
       })
   }, [onDismiss, handleSetError, settingsState, setConfirmationState, tradeContext, onDismissConfirmation, priceImpact])
 
+  useEffect(() => {
+    const unsubOpen = confirmSubscribe(ConfirmEvent.OPEN, () => {
```

Comment:
Now we can fire an event from everywhere (hook, simple functions, components, etc.)